README.md

```diff
@@ -88,7 +88,25 @@ Then, run the following command,
 ```bash
 $ pod install
 ```
+### Carthage
 
+[Carthage](https://github.com/Carthage/Carthage) is  a way to add frameworks to your Cocoa application. You can install it with the following command,
```

Comment:
There are two spaces between `is` and `a way to` :D

Reply:
Additionally, I think we should add Carthage to the table of contents at the top.